pkg/cmd/render/render.go

```diff
@@ -212,9 +221,19 @@ func render() error {
 	return nil
 }
 
+func isDisabledViaCapability(capabilities *v1.ClusterVersionCapabilitiesSpec) bool {
+	if capabilities == nil {
+		return false
+	}
+
+	enabledCaps := sets.New[v1.ClusterVersionCapability](v1.ClusterVersionCapabilitySets[capabilities.BaselineCapabilitySet]...)
```

Comment:
[Installer API][1]:

```go
	// baselineCapabilitySet selects an initial set of
	// optional capabilities to enable, which can be extended via
	// additionalEnabledCapabilities. The default is vCurrent.
	// +optional
	BaselineCapabilitySet configv1.ClusterVersionCapabilitySet `json:"baselineCapabilitySet,omitempty"`
```

I dunno how often folks take advantage of that optionality; at least so far all available caps have been part of `vCurrent`, so it seems unlikely the default is meaningful for anyone.  But you may want to either hard-code `vCurrent` as the default here, or get a knob in openshift/api about what the current default set is, or something, just in case a user leaves `baselineCapabilitySet` unset in their install-config.

[1]: https://github.com/openshift/installer/blob/8dc309c31b1cdc97b383f2514c7ed9ee1828a59c/pkg/types/installconfig.go#L527-L529